airtable/backup_airtable.py

```diff
@@ -2,44 +2,18 @@
 import datetime
 import os
 import tempfile
-import urllib
 import zipfile
 
 from boto.s3.connection import S3Connection
 from boto.s3.key import Key
-import requests
+import airtable
```

Comment:
It should send us emails with errors (right now it just prints them to
stdout).

On Mon, Sep 21, 2015 at 3:55 PM, Mike Perrone <notifications@github.com>
wrote:

> In airtable/backup_airtable.py
> <https://github.com/directactioneverywhere/server/pull/2#discussion_r40037135>
> :
>
> >  import zipfile
> >
> >  from boto.s3.connection import S3Connection
> >  from boto.s3.key import Key
> > -import requests
> > +import airtable
>
> np. It just reads from airtable and writes to s3 with a timestamp, so
> there's no problem with testing it by just running it. You can use the aws
> cli <http://docs.aws.amazon.com/cli/latest/reference/s3/index.html> to
> see if it emitted the backup file:
>
> aws s3 ls s3://dxe-backup/airtable/
>
> You can use the aws credentials that the backup uses for that.
>
> When we integrate this with the server build, we should remember to make
> sure that it still works.
>
> We should think about some way to know if things are broken. Like we could
> have an endpoint on dxetech that tells us the status of all the stuff we
> care about. For the backup this would mean just seeing when the last backup
> was. Not sure what it would mean for the google group sync. I'll add this
> to the agenda.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/directactioneverywhere/server/pull/2/files#r40037135>.
>